### ISSUE TITLE
Kill DatePickerSingle, DatePickerRange.

### ISSUE THREAD

**Issue by tcbegley** [facultyai/dash-bootstrap-components#148]
These two components make our bundle a lot larger through the `react-dates` dependency, but ultimately just replicate functionality present in *dash-core-components*. I think we should kill them and instead encourage use of the *dash-core-components* version, in conjunction with the CSS being added in #147.

Neither component is explicitly documented (though `DatePickerSingle` does appear in the inline form example on the forms page of the docs.). Nevertheless, it would be good to decide on the best way to deprecate these.

**Comment by pbugnion** [comment]
Removing these sounds good. One way to go about removing them could just be to replace them with something like:

```py
class DatePickerSingle(object):

    def __init__(self, *args, **kwargs):
        raise AttributeError("DatePickerSingle has been removed. Use DatePickerSingle from dash-core-components with the attribute className=\"dash-bootstrap\"")
```

Doing this would save an annoying deprecation cycle.

**Comment by tcbegley** [comment]
Resolved by #159
